static/css/main.css

```diff
@@ -7648,8 +7648,9 @@ ul.list_links-minicons li {
     border-width: 0 15px;
     margin-right: -0.25em;
     vertical-align: top;
-    width: 33.33333333%;
+    width: 30%;
```

Comment:
Oops... spoke too soon. You need to make these changes to the widths [in the Less code](https://github.com/cfpb/design-manual/blob/gh-pages/src/static/css/footer.less#L78-L93), lest they be overwritten the next time someone recompiles the Less.

That code currently defines things in terms of grid columns.

Let me know if you need further assistance with updating those.